### ISSUE TITLE
Starting Pods fails: apparmor_parser not found

### ISSUE THREAD

**Issue by hobyte** [k3s-io/k3s#3406]
**Environmental Info:**
K3s Version: 

```bash
# k3s -v
k3s version v1.21.1+k3s1 (75dba57f)
go version go1.16.4
```

Node(s) CPU architecture, OS, and Version: 

```bash
Linux ubuntu-master 4.4.0-210-generic #242-Ubuntu SMP Fri Apr 16 09:57:56 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

Cluster Configuration:

1 Server / single node cluster

**Describe the bug:**

Starting new Pods failed with the error `Error: failed to create containerd container: get apparmor_parser version: exec: "apparmor_parser": executable file not found in $PATH` then after restarting the cluster, all containers fail to start with the same error

**Steps To Reproduce:**

- Installed K3s: 
- configured the k3s system upgrade controller with the stable releases
- Installed [kilo](https://github.com/squat/kilo) (modifies network)

**Expected behavior:**

Pods start without problems

**Actual behavior:**

No pods start

**Additional context / logs:**

[logs](https://pastebin.com/9N6fqc93)

**Comment by brandond** [comment]
Is this actually Ubuntu, or k3os? Please see: https://github.com/rancher/k3os/issues/702

**Comment by hobyte** [comment]
it's ubuntu, but the fix in [rancher/k3os#702](https://github.com/rancher/k3os/issues/702#issuecomment-850246749) fixed the issue
